packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx

```diff
@@ -81,11 +81,9 @@ describe("GIVEN a Grid", () => {
           <ToolkitGridLayoutResponsiveView columns={columns} rows={rows} />
         );
 
-        cy.get(".uitkGridLayout").should(
-          "have.css",
-          "grid-template-columns",
-          "72px 72px 72px 72px 72px 72px 72px 72px 72px 72px 72px 72px"
-        );
+        cy.get(".uitkGridLayout")
+          .invoke("css", "grid-template-columns")
+          .should("match", /^(\d*\.?\d*px *){12}$/);
```

Comment:
🎉

Reply:
This may cause a little confusing for others when first seen

Reply:
agree @origami-z, I tried extracting it to a testColumns() where you could set the number of them but cypress was timing out whenever i tried extracting it :(

Reply:
@origami-z I wasn't scaping  `\`s 🤦‍♀️  pushed a change to clean it :)